.travis.yml

```diff
@@ -1,11 +1,21 @@
 language: node_js
-sudo: false
+sudo: true
 before_install:
   - "npm -g install npm"
+script:
+  - "sudo $(which node) $(which npm) test"
+os:
+  - osx
+  - linux
+matrix:
+  fast_finish: true
+  allow_failures:
+    - os: osx
 node_js:
```

Comment:
I'd be happy to see the version list here updated to match npm itself:
> 4, 6, 0.10, 5, 0.12

in that order